config/settings/base.py

```diff
@@ -80,9 +81,8 @@
     NEW_FRONTEND_DOMAIN = os.environ.get(
         "NEW_FRONTEND_DOMAIN", os.environ.get("PROXY_HOSTNAME_AND_PORT")
     )
-    if not NEW_FRONTEND_DOMAIN.lower().startswith(
-        "http://"
-    ) and not NEW_FRONTEND_DOMAIN.lower().startswith("https://"):
+    if not re.match(r"^https?://", NEW_FRONTEND_DOMAIN, re.IGNORECASE):
```

Comment:
For the record, that was my initial implementation. I opted for a simpler implementation than using regexp as it's something that is not well mastered. From a performance point of view, `startswith` is also more efficient, but considering that we can mix two together, we should measure to confirm which is which.

I'm fine with your changes.